src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -44,6 +44,14 @@ public static ScriptSettingsManager Instance
 
         public virtual bool FileSystemIsReadOnly => IsZipDeployment;
 
+        public bool IsLinuxContainer
+        {
+            get
+            {
+                return !IsAzureEnvironment && !string.IsNullOrEmpty(GetSetting(EnvironmentSettingNames.ContainerName));
```

Comment:
Is there a better env var than CONTAINER_NAME to use to determine this?

Reply:
Wouldn't this also apply to a non-linux container? Should probably add a platform check here: `RuntimeInformation.IsOSPlatform(OSPlatform.Linux)`

Reply:
The CONTAINER_NAME var is something we're only adding in the SeaBreeze scenario, so should be sufficient. The special handling we require for logs (and other things soon) only come from the SeaBreeze host, so I'm looking to use something specific to that environment.